TestAssets/TestProjects/TestAppWithSlnProjectDependencyToRemove/First/First.csproj

```diff
@@ -0,0 +1,7 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
```

Comment:
netstandard2.0? Does not really matter for your tests, just is more inline with what we recommend our customers do when building libraries.

Reply:
Used VS to generate; recommended to be 1.1?

Reply:
Yeah